modules/sagemaker/sagemaker-notebook/setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+license_files =
+  LICENSE
+  NOTICE
+  VERSION
+
+[flake8]
+max-line-length = 120
+extend-ignore = E203, W503
+exclude =
+    .git,
+    __pycache__,
+    docs/source/conf.py,
+    old,
+    build,
+    dist,
+    .venv,
+    codeseeder.out,
+    bundle
+
+[mypy]
+python_version = 3.7
+strict = True
+ignore_missing_imports = True
+allow_untyped_decorators = True
+exclude =
+  codeseeder.out/|example/|tests/|.venv
+warn_unused_ignores = False
```

Comment:
nitpick: This could be included in `pyproject.toml`. However, there's no need to block this PR on this.